draft-ietf-rtcweb-jsep.xml

```diff
@@ -1193,11 +1193,12 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
             any existing candidates will be discarded, and new candidates will
             be gathered from the new servers.
             </t>
-            <t>Any changes to the ICE candidate policy also affect the next
-            gathering phase. If the policy is being made more liberal (i.e.,
-            more permitted candidates), the 'needs-ice-restart' bit will be set.
-            If the policy is being made more strict, no ICE restart is needed,
-            and the bit will not be set. Either way, changes to the policy have
+            <t>Any change to the ICE candidate policy also affects the next
+            gathering phase, and will cause the 'needs-ice-restart' bit to
+            be set. Any candidates which have already been
```

Comment:
I think we can toss the warning altogether since the restart will resolve the situation.